src/context/filter-context.js

```diff
@@ -0,0 +1,21 @@
+import {createContext, useContext, useReducer} from "react"
+import { filterReducer } from "../hooks/filterReducer";
+
+const FilterContext = createContext();
+
+const FilterProvider = ({children}) => {
+    const [state, dispatch] = useReducer(filterReducer, {
+        priceSort: "",
+        category: [],
+        ratingSort: "",
+        range: 2000
```

Comment:
why are you initializing range from 2000??
is your products price starts from 2000???

Reply:
no, that's the max price. without applying any filters all the products under 2000 will be shown.